src/styles/blocks/clock.scss

```
@@ -0,0 +1,51 @@
+.clock {
+  position: relative;
+  height: $clock--size;
+  width: $clock--size;
+
+  background: no-repeat url("../img/clock.png");
+  background-size: cover;
+
+  &__center {
+    @include position(
+      $top: 50%,
+      $right: unset,
+      $bottom: unset,
+      $left: 50%
+    );
```

if you apply my changes in mixin scss file, you can use it like
```suggestion
    @include position() {
      top: 50%;
      left: 50%;
    };
```